packages/kbn-test/src/functional_test_runner/run_check_ftr_code_owners.ts

```diff
@@ -35,12 +35,9 @@ export async function runCheckFtrCodeOwnersCli() {
       const reversedCodeowners = getPathsWithOwnersReversed();
 
       const testFiles = await getRepoFiles(TEST_DIRECTORIES);
-      for (const { repoRel } of testFiles) {
-        const owners = getCodeOwnersForFile(repoRel, reversedCodeowners);
-        if (owners === undefined || owners === '') {
```

Comment:
We should not remove the check for an empty string - it has been introduced intentionally and is still required as we actually have entries with an empty owner string and we want to treat them as missing owners for the purpose of this script.